app/views/tweets/tweets.erb

```diff
@@ -0,0 +1,5 @@
+Welcome, dese r tweets.
+
+<%Tweet.all.each do |tweet|%>
```

Comment:
Convention would be to access all the tweets in the controller and save them in an instance variable

```ruby
#controller
@all_tweets = Tweet.all
```

Here in your view you would use the instance variable 

```ruby
<% @all_tweets.each do |tweet| %>
```